fuzz-tests/fuzz.sh

```diff
@@ -170,8 +170,8 @@ function fuzzer_simple() {
     fi
 
     set -x
-    cargo $cmd --release \
```

Comment:
Why was the `--release` flag removed here?

Reply:
To get more debug information when using `simple_fuzzer` to reproduce some crashes.
Just thinking that maybe I could allow to specify it. 
Would be useful for producing code coverage - should perform faster 🤔

Reply:
Restored `--release` flag. Getting code coverage is way faster when in release mode.